src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,7 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: axesAttrs.visible,
```

Comment:
Perhaps the right answer is not to remove the attribute and avoid supplying that defaults in that case.